recipes/hydra/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "hydra" %}
```

As there are quite a lot of packages called "hydra" would this be more suitable?
```suggestion
{% set name = "facebook-hydra" %}
```

---

I don't know that this is a blocker for me

---

If a name change is required, I would rather suggest something more related to its purpose, like "hydra-config".
@omry, do you have any opinion on this? By the way, should I add you as a feedstock maintainer and can you help with the CI issues?

---

> If a name change is required, I would rather suggest something more related to its purpose, like "hydra-config".
> @omry, do you have any opinion on this? By the way, should I add you as a feedstock maintainer and can you help with the CI issues?

hydra-core, to match the pip package name (same problem in pypi).

I have no experience maintaining conda packages.
Happy to help with any issues as long as you own the process.
I can also update the documentation to reflect that conda install is a supported installation method once this land.

---

@mdraw, can you join the Hydra chat? we can talk about the failure.